files/client.ini

```diff
@@ -0,0 +1,3 @@
+[client]
```

Comment:
Can we use `.json` as the config type because it allows for deeper nesting, lists, and also ops tools have nice functions for mangling/merging JSON?

Reply:
can we stick with ini for simplicity for now?

Reply:
sure